test/nodes/test_multi_node_weighted_sampler.py

```diff
@@ -154,49 +149,15 @@ def test_multi_node_weighted_sampler_cycle_until_all_exhausted(self) -> None:
             self.assertEqual(sorted(datasets_in_results), ["ds0", "ds1", "ds2", "ds3"])
             mixer.reset()
 
-    @parameterized.expand(
-        itertools.product(
-            [1, 4, 7],
-            [
-                StopCriteria.ALL_DATASETS_EXHAUSTED,
-                StopCriteria.FIRST_DATASET_EXHAUSTED,
-                StopCriteria.CYCLE_UNTIL_ALL_DATASETS_EXHAUSTED,
-            ],
-        )
-    )
-    def test_save_load_state_mixer(self, midpoint: int, stop_criteria: str):
-        mixer = MultiNodeWeightedSampler(self.datasets, self.weights, stop_criteria)
-        run_test_save_load_state(self, mixer, midpoint)
```

Comment:
This test is already covered, see `test_save_load_state_mixer_over_multiple_epochs`